WcaOnRails/app/helpers/application_helper.rb

```diff
@@ -53,12 +53,12 @@ def anchorable(pretty_text, id = nil)
   WCA_EXCERPT_RADIUS = 50
 
   def wca_excerpt(html, phrases)
```

Comment:
Are there any actual changes to this method?

Reply:
As discussed with @jonatanklosko offline, yes there are =) Previously the `text` variable was transliterated, but now it's not, which means the final string will have the non transliterated characters (as desired).